src/utils/logger.js

```diff
@@ -0,0 +1,34 @@
+import fs from 'fs';
+import winston from 'winston';
+import 'winston-daily-rotate-file';
+
+const logDir = process.env.LOGGING_DIR || 'logs';
+const logLevel = process.env.LOGGING_LEVEL || 'info';
+const tsFormat = new Date().toLocaleTimeString();
+
+// Create log directory if it does not exist
+if (!fs.existsSync(logDir)) {
+  fs.mkdirSync(logDir);
+}
+
+/**
+ * Create new winston logger instance.
+ */
+const logger = new (winston.Logger)({
+  transports: [
+    new winston.transports.Console({
+      timestamp: tsFormat,
+      colorize: true,
+      level: 'info'
+    }),
+    new winston.transports.DailyRotateFile({
+      filename: `${logDir}/-debug.log`,
+      timestamp: tsFormat,
+      datePattern: 'yyyy-MM-dd',
+      prepend: true,
+      level: logLevel
+    })
+  ]
+});
+
```

Comment:
@badyakarprabin I think you've made some changes in this log format in the chill right? Copy that from here https://github.com/leapfrogtechnology/chill.